### DISCUSSION TITLE
Action on session timeout

### DISCUSSION THREAD

**Discussion by OlivierBOEL** [OpenIDC/mod_auth_openidc#689] [Q&A]
Hello,

AFAIK, if session expires (OIDCSessionInactivityTimeout or OIDCSessionMaxDuration), user is re-directed to OIDCProviderAuthorizationEndpoint.
Is there any way to avoid and re-direct to some configurable page that clearly informs the user that his session has expired?

Thanks,

Olivier

**Comment by zandbelt** [comment]
no, there's currently no way to do that
